proposal/abc225666/README.md

```diff
@@ -0,0 +1,75 @@
+# Agar backend api
+
+[![hackmd-github-sync-badge](https://hackmd.io/RYp6o8DmTr2QNHb7f-hbCg/badge)](https://hackmd.io/RYp6o8DmTr2QNHb7f-hbCg)
+
+
+## Basic information
+Github repository: https://github.com/abc225666/agar-backend-api
+
+## Problem to solve
+Agar is mini game produced by agar.io. People control a circle to competitive with other people. The larger your circle is, the more point you get and your circle is more powerful.
```

Comment:
The game employs a simple algorithm and it doesn't seem to utilize significant amount of computation.  The kinetics does seem to involve real-number calculations.